internal/healthchecks/network_check.go

```diff
@@ -20,9 +20,38 @@ import (
 )
 
 var (
-	// API urls
-	loggingAPIUrl    = "https://logging.googleapis.com/$discovery/rest"
-	monitoringAPIUrl = "https://monitoring.googleapis.com/$discovery/rest"
+	requests = [...]networkCheckRequest{
+		{
+			name:             "Logging API",
+			url:              "https://logging.googleapis.com/$discovery/rest",
+			successMessage:   "Request to the Logging API was successful.",
+			healthCheckError: LogApiConnErr,
+		},
+		{
+			name:             "Monitoring API",
+			url:              "https://monitoring.googleapis.com/$discovery/rest",
+			successMessage:   "Request to the Monitoring API was successful.",
+			healthCheckError: MonApiConnErr,
+		},
+		{
+			name:             "Packages API",
+			url:              "https://packages.googleapi.com/$discovery/rest",
```

Comment:
Although I don't mind that much adding these urls to the health checks, they are more valuable at the diagnostics script and don't add much value after the agent is already installed. Since they refer to the endpoints that will enable the installation to succeed, without access to packages and/or dl.google you can't download the Ops-Agent and run the health checks.

Reply:
This is a great catch that I haven't realized before. It doesn't hurt to check them here, but there is more value in the diagnostics script if the user can't even install the Ops Agent.

Reply:
Checking the `metadata` server here still has a lot of value.